### ISSUE TITLE
remove need for merged=True from user interface

### ISSUE THREAD

**Issue by nicolasaunai** [PHAREHUB/PHARE#849]
Users should not have to worry about having to set merge=True or not when getting data with `run.GetXXX()`.
Merged=True is currently there so that users may get an interpolator to get data wherever they want from the finest data in the hierarchy. They should be able to do that from a `PatchHierarchy` already. There's the `__call__()` in `PatchHierarchy` but I think it's kinda broken as it's (and was intended as is) not using an interpolator, but that probably is silly...

**Comment by PhilipDeegan** [comment]
is this still the case since recent PatchHierarchy updates?

**Comment by nicolasaunai** [comment]
yes, it's still there and I think it should not. The idea is that one should be able to do a slice in a PatchHierarchy representing a ScalarField or a VectorField anywhere and that should do the interpolation as needed without the need for the user to a priori decide whether to put merged=True or not. There should only be PatchHierarchies and that's it

**Comment by PhilipDeegan** [comment]
the initial work to close this is done in #1195
once that's in, we should be able to remove "merged=" entirely in a second follow up PR
